Makefile

```diff
@@ -125,6 +125,9 @@ GO_GCFLAGS=$(shell				\
 
 BINARIES=$(addprefix bin/,$(COMMANDS))
 
+# Forcibly set the default goal to all, in case an include brings in a rule definition.
+.DEFAULT_GOAL := all
```

Comment:
Think this is clearer next to the other goal definitions down below.

Reply:
Moved it down by .PHONY and all